client/styles/SettingsPage.module.css

```diff
@@ -0,0 +1,54 @@
+.settings-form-container {
+  padding: 20px 16px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.settings-page-title {
+  margin-bottom: 16px;
+  align-self: center;
+  font-size: 40px;
+}
+
+.settings-field-group {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  align-items: center;
+}
+.settings-field-label {
+  text-align: left;
+  align-self: flex-start;
+  font-size: 20px;
+}
+
+.settings-text-field :global(.MuiOutlinedInput-root) {
+  border-radius: 10px;
+  width: 300px;
+  border: 2px solid var(--secondary-main);
+}
+
+.settings-submit-btn {
```

Comment:
Maybe we can make this button styling more aligned with the rest of the buttons? In particular, I find the border radius and size quite different from other buttons. An alternative solution is using our `useDebounce` hook to auto-save the new name so we won't need the button (only if it is easy to implement).